packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -74,11 +74,7 @@ $Breadcrumb-itemMaxWidth-sm: 116px;
   vertical-align: top;
 }
 
-.ms-Breadcrumb-itemLink,
-.ms-Breadcrumb-overflowButton {
-  text-decoration: none;
-  outline: transparent;
-
+a.ms-Breadcrumb-itemLink {
```

Comment:
why apply the `itemLink` class at all? wouldn't it make more sense to have something like `-item` and then if its clickable to also apply `-itemLink`?